mlir/include/mlir/Dialect/Rock/IR/RockConvInterface.td

```diff
@@ -43,6 +43,13 @@ def RockConvInterface : OpInterface<"RockConvInterface"> {
         /*args=*/(ins),
         /*methodBody=*/"",
         /*defaultImplementation=*/[{ return $_op.getDilations(); }]>,
+    InterfaceMethod<
+        /*desc=*/[{Return features attr values.}],
+        /*retType=*/"GemmFeatures",
+        /*methodName=*/"getFeatures",
+        /*args=*/(ins),
+        /*methodBody=*/"",
+        /*defaultImplementation=*/"">,
```

Comment:
I think you should actually set a default implementation here.

Reply:
Ideally, having default impl here makes better sense but getFeatures()  per  ops  are all automatically generated by the tablegen.